recipes/polyglotdb/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "polyglotdb" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 89d2b4a19595a7bc14d1fba124afc63454baa9678275cd093d091d0430fc36c2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - conch_sounds
+    - influxdb
+    - librosa
+    - neo4j-python-driver
+    - pip
+    - praatio
+    - python
+    - requests
+    - textgrid
+    - tqdm
+  run:
+    - conch_sounds
+    - influxdb
+    - librosa
+    - neo4j-python-driver
+    - praatio
+    - python
+    - requests
+    - textgrid
+    - tqdm
```

Comment:
Please add the version requirements here from upstream.

Reply:
- Version requirements as lower bounds acceptable? (version >=, = x.x ?)
- Should I put these requirements for all dependencies in this PR?

Thanks for the feedback by the way!

Reply:
Yes that's fine. You should put all required dependencies and their version constraints here. I would recommend using https://github.com/conda-incubator/grayskull it's normally pretty good at detecting everything automatically.

Reply:
Sounds good, my template came from grayskull so I'll manually fill in the rest. Thanks again